js/search/searchUtils.js

```diff
@@ -22,7 +22,7 @@ class SearchUtils {
         return new Promise((resolve, reject) => {
             if (!isMac) {
                 try {
-                    this.path = this.path.substring(0, 2);
+                    this.path = this.path.substring(0, 1);
```

Comment:
Please add code comments explaining what this does.

Reply:
Included in the code documentation